spectree/plugins/falcon_plugin.py

```diff
@@ -167,27 +170,59 @@ def parse_path(self, route, path_parameter_descriptions):
 
         return f'/{"/".join(subs)}', parameters
 
-    def request_validation(self, req, query, json, headers, cookies):
+    def parse_field(self, field):
+        if isinstance(field, list):
+            return [self.parse_field(subfield) for subfield in field]
+
+        # When file name isn't ascii FieldStorage will not consider it.
+        encoded = field.disposition_options.get("filename*")
+        if encoded:
+            encoding, filename = encoded.split("''")
+            field.filename = filename
+            field.file = BytesIO(field.file.read().encode(encoding))
+        if getattr(field, "filename", False):
```

Comment:
I think `getattr(field, "filename", False)` will always return `None` since the type of `field` is always `cgi.FieldStorage`.